gui/src/app/pages/HomePage/HomePage.tsx

```diff
@@ -16,14 +18,15 @@ type Props = {
 const defaultStanContent = ''
 const defaultDataContent = ''
 const defaultMetaContent = '{"title": "Untitled"}'
+const defaultSamplingOptsContent = JSON.stringify(defaultSamplingOpts, null, 2)
```

Comment:
Is it possible to manually inspect this json? Just wondering about manually setting the indent spacing.

Reply:
It's not visible to the user. I have removed the `null, 2` part